python/cudf/cudf/bindings/issorted.pyx

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+from cudf.bindings.cudf_cpp cimport *
+from cudf.bindings.cudf_cpp import *
+from cudf.bindings.issorted cimport *
+from cudf.bindings.utils cimport *
+
+cpdef issorted(columns, descending, nulls_are_smallest=False):
+    """
+    Checks whether the rows of a `table` are sorted in a lexicographical order.
+
+    Parameters
+    ----------
+    columns : list of columns
+    descending : array of order of columns, 0 - ascending, 1 - decending
+                 if array passed is of zero length, by deafult all columns
+                 are considered ascending. If this an empty vector, then it
+                 will be assumed that each column is in ascending order.
+    nulls_are_smallest : True indicates nulls are to be considered
+                         smaller than non-nulls ; false indicates oppositie
+
+    Returns
+    -------
+    result : True - if sorted; False - if not.
+    """
+
+    cdef cudf_table *c_values_table = table_from_columns(columns)
+    cdef vector[int8_t] c_descending_vector = descending
```

Comment:
Should we specify a default value of `descending` here? If it's unset does Cython initialize it to NULL by default?

Reply:
We are expecting an empty list, which is nothing but a 0 sized vector.

Reply:
I can check if the length is zero/null and add default value to it.

Reply:
How about this ?
`if (descending == None):
        descending = []
cdef vector[int8_t] c_descending_vector = descending`

Reply:
I'm thinking we just add a default value to the function definition of an empty list and call it a day to guard against undefined behavior.

Reply:
That's better.